CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Added
+* Batch insert/upsert operation
+  `crud.insert_many()`/`crud.insert_object_many()`/
+  `crud.upsert_many()`/`crud.upsert_object_many()`
```

Comment:
Didn't we decided to not implement `upsert` since we didn't want to rework its API now? It would be really bad to release it with current API and change it in the next release.

Reply:
The upsert_many API was changed.